app.js

```diff
@@ -298,5 +342,16 @@ function run_extension_functions() {
 
 // setTimeout(change_youtube_logo_2_loading, 1000);
 
-setTimeout(run_extension_functions, 1000); // Delay of 1000 = 1 seconds
+function loadCheck() {
+    if (document.getElementsByClassName('mobile-topbar-header-endpoint')) {
+        run_extension_functions()
+    } else {
+        setTimeout(loadCheck, 15);
+    }
+}
 
+loadCheck()
```

Comment:
just to be better readable maybe change the function name to check_page_elements_loaded() or something in those lines so if someone is reading the code could easily understand what this function is doing

Reply:
Thanks